common/src/types/token.rs

```diff
@@ -626,6 +626,28 @@ lazy_static! {
             ),
         ].into_iter().collect()
     };
+
+    /// Remaps Katana ERC20 addresses to their Ethereum mainnet counterparts
+    /// so that a price can be determined
+    ///
+    /// deployments.json generated by renegade-contracts
+    ///
+    /// TODO: This will be removed when we implement our custom bridging solution
+    #[derive(Debug)]
```

Comment:
shouldn't need this derive here, hashmap [already implements `Debug`](https://doc.rust-lang.org/stable/std/collections/struct.HashMap.html#impl-Debug-for-HashMap%3CK,+V,+S%3E)